build/test_metadata_utilities.py

```diff
@@ -0,0 +1,54 @@
+import metadata_utilities
```

Comment:
We should call this as part of the build

Reply:
Done, too includes it now.